content/docs/introduction/billing-azure-marketplace.md

```diff
@@ -0,0 +1,61 @@
+
+## Create a Neon Resource
+
+1. On the **Create a Neon Resource** page, set the following values in the **Create Neon Resource** section.
+
+   | Property                   | Description                                                                                                                                                                                                                                                                                                        |
+   | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+   | **Subscription**           | From the drop-down, select your Azure subscription where you have Owner or Contributor access.                                                                                                                                                                                                                     |
+   | **Resource group**         | Specify whether you want to create a new resource group or use an existing one. A resource group is a container that holds related resources for an Azure solution. For more information, see [Azure Resource Group overview](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/overview). |
+   | **Resource Name**          | Enter the name for the Neon organization you want to create.                                                                                                                                                                                                                                                       |
+   | **Region**                 | Select a region to deploy your resource. Neon Serverless Postgres will let you choose a region while creating a project.                                                                                                                                                                                           |
+   | **Neon Organization name** | Corresponds to the name of your company, usually.                                                                                                                                                                                                                                                                  |
+   | **Pricing Plan**           | Choose a plan of your choice. For information about Neon's plans, please refer to the [Neon Pricing](https://neon.tech/home) page. Neon offers a Free Plan and paid plans (**Scale**, and **Business**) that you can choose from.                                                                                  |
+
+    <Admonition type="note">
+    The Neon **Launch Plan** is currently not supported.
```

Comment:
"Currently not available in the Azure Marketplace".

Reply:
fixed.